R/pkg/tests/fulltests/test_utils.R

```diff
@@ -89,7 +89,10 @@ test_that("cleanClosure on R functions", {
     lapply(x, g) + 1  # Test for capturing function call "g"'s closure as a argument of lapply.
     l$field[1, 1] <- 3  # Test for access operators `$`.
     res <- defUse + l$field[1, ]  # Test for def-use chain of "defUse", and "" symbol.
-    f(res)  # Test for recursive calls.
+    # Enable once SPARK-30629 is fixed
```

Comment:
Does this fail with `testthat >= 2.0.0`?

Reply:
@dongjoon-hyun It fails on any version, but is somehow silenced if we use testthat 1.x. I've already opened a JIRA ticket for ‒ [SPARK-30629](https://issues.apache.org/jira/browse/SPARK-30629). Can be easily reproduced outside tests.

Reply:
Let's fix SPARK-30629 first.

Reply:
OK. Set SPARK-30629  as a blocker for SPARK-23435.

Reply:
Thanks, @zero323 . SPARK-30629 only happens at test that >= 2.0.0. It should be a blocker for this (SPARK-23435).

Reply:
Dropping this commit from here.

Reply:
CC @falaki